.travis.yml

```diff
@@ -1,46 +1,57 @@
 sudo: false
 language: python
-python:
- - "2.7"
- - "3.3"
- - "3.4"
- - "3.5"
- - "3.6"
+
+python: "3.6"
+
 env:
   matrix:
-   - DJANGO="Django<1.9"
```

Comment:
Okay, same comment as in previous pull request: Python 3.3 with Django 1.8 is still supported until September, 2017, you can check it in version history table here https://en.wikipedia.org/wiki/CPython or directly in https://www.python.org/dev/peps/pep-0398/#lifespan ...

I hope this pull request will be merged before September, 2017 :-)))